src/package.json

```
@@ -55,6 +55,23 @@
     "bootstrap": "^3.3.7",
     "d3": "^5.9.2"
   },
+  "exports": {
+    "./styles/*": {
+      "style": "./styles/*.css"
+    },
+    "./less/ux-aspects.less": {
+      "style": "./less/ux-aspects.less"
+    },
+    "./fonts/*": {
+      "asset:": "./fonts/*"
+    },
+    "./img/*": {
+      "asset": "./img/*"
+    },
+    "./icons/ux/*": {
+      "asset": "./icons/ux/*"
+    }
+  },
```

I did a bit of playing about with this, we need to include "default" properties on these to allow them to be imported from a JS/TS file as well as from a stylesheet (note we need them to be separate entries for each file in that instance).

I also think we can omit the asset files for now

```suggestion
    "./styles/ux-aspects.css": {
      "style": "./styles/ux-aspects.css",
      "default": "./styles/ux-aspects.css"
    },
    "./styles/ux-aspects.min.css": {
      "style": "./styles/ux-aspects.min.css",
      "default": "./styles/ux-aspects.min.css"
    },
    "./less/ux-aspects.less": {
      "style": "./less/ux-aspects.less",
      "default": "./less/ux-aspects.less"
    },
```